src/workflowyfx.js

```diff
@@ -45,6 +45,14 @@ $(`<button accesskey="n">`)
 		return false
 	})
 
+var modsHtml;
+var mods = accessKeyModifiers(navigator.userAgent)
+if (mods) {
+	modsHtml = `<kbd>${mods.join('</kbd> + <kbd>')}</kbd> + `
```

Comment:
The `kbd` made me curious. I thought you might've picked them up from the original source, so I went looking there. Look what I found:

```html
<td class="commandDescription">
	<span class="showOnWindows showOnLinux">Alt</span>
	<span class="showOnChromeOS">Control</span>
	<span class="hideOnWindows hideOnLinux hideOnChromeOS">⌘</span> + Shift + Up/Down
</td>
```

I'd like to stay close to original so unless you think this is a terrible idea, I'd want to go that way. Would eliminate any browser sniffing code from the extension (for now), which is not bad either. What do you think?